src/browser/modules/Stream/Auth/ConnectForm.tsx

```diff
@@ -28,12 +28,21 @@ import {
   StyledConnectionLabel,
   StyledConnectionFormEntry,
   StyledSegment,
-  StyledBoltUrlHintText
+  StyledBoltUrlHintText,
+  FormContainer,
+  SsoOptions,
```

Comment:
Consistent capitalization would be nice here :-)

Reply:
Would be good to do a search/replace for all `sso` and `Sso` and use `SSO` consistently